package.json

```diff
@@ -13,10 +13,10 @@
   "devDependencies": {
     "bower": "1.3.8",
     "chai": "1.10.0",
-    "jshint": "2.5.2",
+    "jshint": "2.8.0",
```

Comment:
Usually, but in this case I don't want jshint dependency to move around at all, lest its behavior suddenly changes, causing failures in CI for unsuspecting contributors and their pull requests.